### ISSUE TITLE
Move all handlers from core in a new package handler?

### ISSUE THREAD

**Issue by decebals** [decebals/pippo#138]
A new package `core.handler` and move all handlers (*Handler.java, route.*Handler.java) in this package.

**Comment by gitblit** [comment]
I agree, the *core* package is a little fat.  Another alternative would be moving `*Handler.java` into *core.route*?  We already have that and they are directly related - a *Route* has a url, method, and a Handler.  That would add 5-6 classes to *core.route* perhaps making it too fat.  If we a create a new package that moves 5-6 classes out of *core* and 5-6 classes out of *core.route*.  I'm fine with whatever you think is best.

Unrelated, but looking at the *Route* constructor I'm struck by feeling the signature is backwards:

```java
public Route(String uriPattern, String requestMethod, RouteHandler routeHandler)
```

I think everywhere else we specify the order as METHOD URL (.e.g. GET /webjars/jquery...)
